strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationFileManager.java

```diff
@@ -36,6 +36,7 @@ public void store(final Configuration configuration)
         try
         {
             Resource configurationResource = getConfigurationResource();
+            //Check that target resource stored on FS and not under classpath for example
             if (!configurationResource.isFile())
             {
                 LOGGER.warn(String.format("Skip configuration resource store [%s]", configurationResource));
```

Comment:
Hey, would you mind changing the name of this logger to lowercase? Everything else looks great and you can proceed with self-merging it after this minor change request.

Reply:
ok

Reply:
Thanks!